isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/referenzarchitektur/inhalt.adoc

```diff
@@ -468,6 +468,45 @@ Für derartige Aufrufe gibt es keine Vorgaben: Sie können aus beliebigen Schich
 [id="image-CallFAmGuX",reftext="{figure-caption} {counter:figures}"]
 image::blaupausen:referenzarchitektur/CallFAmGuX.png[align="center"]
 
+[[zentrale-sicherheitsaspekte]]
+=== Zentrale Sicherheitsaspekte in der Referenzarchitektur
+
+Die IT-Sicherheit ist insbesondere dann für Softwaresysteme eine wichtige Anforderung, wenn diese sensible oder gar personenbezogene Daten verarbeiten.
+Solche Systeme stellen hohe Anforderungen an Vertraulichkeit, Integrität und Verfügbarkeit.
+Eine Referenzarchitektur muss als grundlegendes Element der IT-Sicherheitsstrategie die Einhaltung solcher Schutzziele unterstützen.
+
+Ein Pfeiler der IT-Sicherheitsstrategie ist der Schutz der Daten und IT-Systeme vor unberechtigter Nutzung.
```

Comment:
Welche IT-Sicherheitsstrategie? Vom BVA? Wenn nicht referenzierbar, dann bitte die Zeile entfernen.